test/saithrift/tests/sail3.py

```diff
@@ -941,6 +956,7 @@ def runTest(self):
         addr_family = SAI_IP_ADDR_FAMILY_IPV4
         ip_addr1 = '10.10.0.0'
         ip_mask1 = '255.255.0.0'
+        ip_mask2 = '255.255.255.255'
```

Comment:
change to /24 for all direct reachable route. It's a more common use case. We are not yet to test corn cases here.